scripts/state_machine.py

```diff
@@ -30,7 +30,7 @@ class StateMachine:
     >>> StateMachine([('ab','a','b','a')]).final_state('ab')
     'b'
     >>> StateMachine([('ab','a','b','a')]).transition_regex('ab')
-    re.compile('a$')
+    re.compile('a$', re.IGNORECASE)
```

Comment:
Changes in this file also seem to mirror #493 but I don't see an obvious relationship between the branches since dc6458e5ef8cd745293b4f3de3c54b34239ef137.
Might make merging later more fun?

Reply:
Hopefully those changes were cherry-picked or the two PRs started from the same branch, in which case there won't be any merge conflicts. But in any case #493 should be merged first, then the update from feature/capgen pulled into this branch, and these diffs should go away magically.

Reply:
@climbfuji is correct. I had included them in the cleanup PR but then needed the updates in this PR to get the tests to pass. After I merge #493  I will confirm this diffs go away.